ethpm_types/contract_type.py

```diff
@@ -46,6 +46,74 @@ class ContractInstance(BaseModel):
     runtime_bytecode: Optional[Bytecode] = Field(None, alias="runtimeBytecode")
 
 
+class SourceMapItem(BaseModel):
+    # NOTE: `None` entry means this path was inserted by the compiler during codegen
```

Comment:
I don't see any `None` here, what exactly is a `None` entry, just non-initialized or is this comment not needed

Reply:
Oh I see, like when the object gets initialized